search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeService.kt

```diff
@@ -56,6 +56,7 @@ class TemporalEntityAttributeService(
             .bind(
                 "entity_payload",
                 temporalEntityAttribute.entityPayload?.let { Json.of(temporalEntityAttribute.entityPayload) })
+            .bind("dataset_id", temporalEntityAttribute.datasetId)
```

Comment:
as it is an Uri shouldn't it be converted to a string ?

Reply:
The cast isn't required.